language/tools/move-cli/src/sandbox/utils/mod.rs

```diff
@@ -98,6 +100,41 @@ fn print_struct_with_indent(value: &AnnotatedMoveStruct, indent: u64) {
     }
 }
 
+// Print a struct with a specified outer indent
+fn print_struct_diff_with_indent(
+    value1: &AnnotatedMoveStruct,
+    value2: &AnnotatedMoveStruct,
+    indent: u64,
+) {
+    let indent_str: String = (0..indent).map(|_| " ").collect::<String>();
+    let prev_str = format!("{}", value1);
+    let new_str = format!("{}", value2);
+
+    let Changeset { diffs, .. } = Changeset::new(&prev_str, &new_str, "\n");
+
+    let mut t = term::stdout().unwrap();
+
+    for diff in diffs {
+        match diff {
+            Difference::Same(ref x) => {
+                t.reset().unwrap();
+                let _ = writeln!(t, " {}{}", indent_str, x);
```

Comment:
Nit: you should probably `unwrap` these to make sure that the result is used
```suggestion
            writeln!(t, " {}{}", indent_str, x).unwrap();
```